gazebo/create_dockerfiles.py

```diff
@@ -16,6 +19,35 @@
 from ros_buildfarm.docker_common import OrderedLoad
 
 
+def checkVersion(data):
+    """Checks current package_version from online package index"""
+
+    # Determine URL
+    urlTemplate = string.Template(
+        """
+        http://packages.osrfoundation.org/gazebo/$os_name/dists/$os_code_name/main/binary-$arch/Packages
```

Comment:
It just works but I would prefer to use the `$os_name-stable` to name the stable repositories. Minor thing.